integration/container/daemon_linux_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/integration-cli/daemon"
 	"github.com/docker/docker/integration/internal/container"
+	"github.com/gotestyourself/gotestyourself/skip"
```

Comment:
How is the library useful compared to plain `if foo { t.Skip(bar} }`?

Reply:
I think it makes the skip message more consistent.

https://godoc.org/github.com/gotestyourself/gotestyourself/skip

Reply:
@AkihiroSuda 

Because it guarantees the skip message are:
1. Consistent
2. Complete
3. Correct

That example is not an accurate comparison. If you are skipping based on some condition, then the error message should always include that condition, and optionally more details about why that condition is relevant to that test case.

Your example leads to this:

```go
if someCondition {
    t.Skip("some text that is completely unrelated to the condition, like I feel like it")
}
```

That skip message is not helpful and could be misleading. It is missing the actual condition that evaluated to true.

With `skip.If()`:

```
SKIP: !testEnv.IsLocalDaemon(): cannot start daemon on remote test run
```